src/styles/blocks/reviews.scss

```diff
@@ -0,0 +1,13 @@
+.reviews {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+
+  &__text {
+    margin: 0;
```

Comment:
it's better to reset it inside `_reset/_normalize.scss` for list of tags you needed